core/src/test/java/org/elasticsearch/gateway/MetaDataStateFormatTests.java

```diff
@@ -252,9 +251,10 @@ public void testLatestVersionDoesNotUseLegacy() throws IOException {
         for (int i = 0; i < numLegacyFiles; ++i) {
             final Path dir2 = randomFrom(dirs);
             final int v2 = v1 + 1 + randomInt(10);
-            try (XContentBuilder xcontentBuilder = XContentFactory.contentBuilder(format.format(), Files.newOutputStream(dir2.resolve(MetaDataStateFormat.STATE_DIR_NAME).resolve(MetaStateService.GLOBAL_STATE_FILE_PREFIX + v2)))) {
+            try (XContentBuilder xcontentBuilder = XContentFactory.contentBuilder(format.format(),
+                Files.newOutputStream(dir2.resolve(MetaDataStateFormat.STATE_DIR_NAME).resolve(MetaData.GLOBAL_STATE_FILE_PREFIX + v2)))) {
                 xcontentBuilder.startObject();
-                MetaData.Builder.toXContent(randomMeta(), xcontentBuilder, params);
```

Comment:
why not use the Builder.toXContent directly, which is what we used to do?  I'm not sure the test is relevant at all for 5.0. I think it tests for the transition from 1.x to 2.0?